modules/vba_documents/README.yml

```diff
@@ -387,6 +387,7 @@ components:
             * `received` - Indicates document package has been successfully propagated to Central Mail API
             * `processing` - Indicates document package is being processed by ICMHS or a downstream system.
             * `success` - Indicates document package has been received by DHMS.
+            * `vbms success` - Indicates that the the package has been succesfully processed into VBMS.
```

Comment:
All of the statuses are singularly worded, should we consider vbms_success possibly?

Reply:
Remove